contracts/governance/extensions/GovernorTimelockAccess.sol

```diff
@@ -24,7 +24,7 @@ import {Time} from "../../utils/types/Time.sol";
  *
  * ==== Security Considerations
  *
- * Some operations may be cancelable in the `AccessManager` by the admin or a set of guardians, depending on the
+ * Some operations may be cancellable in the `AccessManager` by the admin or a set of guardians, depending on the
```

Comment:
See the comment below. He we are going in the opposite direction. We shouldn't just have these flip-flopping.

Reply:
We don't have much consistency throughout the repo. We should chose one and stay with it. I prefer American spelling.